pom.xml

```diff
@@ -1445,6 +1445,12 @@
                 <groupId>com.esotericsoftware.kryo</groupId>
                 <artifactId>kryo</artifactId>
                 <version>2.21</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.ow2.asm</groupId>
+                        <artifactId>asm</artifactId>
```

Comment:
is asm still on the classpath at all, even in test scope? Seems like kryo might actually need asm, but maybe not in the way we use it.